modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -104,12 +91,10 @@ function openToast({
 	container,
 	containerId,
 	message = '',
-	messageType = TYPES.TEXT,
 	onClick = () => {},
 	onClose = () => {},
 	renderData = DEFAULT_RENDER_DATA,
-	title,
-	titleType = TYPES.TEXT,
+	title = '',
```

Comment:
Ugh, I added this initially before I checked if it existed... we can get rid of this.